src/pages/payments/payments-list/ui.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-unused-vars */
+import React, { useCallback } from 'react'
+import { styled } from '@ui/theme'
+import { PaymentsStackParamList } from '@processes/routing/types'
+import { PaymentsFlatList } from '@features/payments-list'
+import { PaymentServiceUI } from '@shared/api/payment-categories'
+import { usePaymentsCategories } from '@entities/payments-categories'
+
+import { configFlatListItems } from './lib'
+
+const Wrapper = styled.SafeAreaView`
+  background: ${({ theme }) => theme.palette.background.secondary};
+  flex: 1;
+  display: flex;
+  gap: 20px;
+  padding: 18px;
+`
+
+const CategoriesFlatListWrapper = styled(PaymentsFlatList)`
+    top: 20px;
+`
+
+type Props = {
+    navigateTo(screenName: keyof PaymentsStackParamList, title?: string, services?: PaymentServiceUI[]): void
```

Comment:
```suggestion
    submit: (title?: string, services?: PaymentServiceUI[]) => void
```

Не нужно подтягивать сюда логику навигации. Передавай только те параметры которые тянуться из вью
route должен остаться на уровне навигации